README.md

```diff
@@ -506,6 +514,60 @@ By default, the Klaviyo SDK automatically clears the badge count on app open. If
 
 Klaviyo SDK will automatically handle the badge count associated with Klaviyo pushes. If you need to manually update the badge count to account for other notification sources, use the `KlaviyoSDK().setBadgeCount(:)` method, which will update the badge count and keep it in sync with the Klaviyo SDK. This method should be used instead of (rather than in addition to) setting the badge count using `UNUserNotificationCenter` and/or `UIApplication` methods.
 
+## In-App Forms
+> ℹ️ In-app forms support is available in SDK version [4.2.0](https://github.com/klaviyo/klaviyo-swift-sdk/releases/tag/4.2.0) and higher
+
+[In-app forms](https://help.klaviyo.com/hc/en-us/articles/34567685177883) are messages displayed to mobile app users while they are actively using an app. You can create new in-app forms in a drag-and-drop editor in the Sign-Up Forms tab in Klaviyo.  Follow the instructions in this section to integrate forms with your app. The SDK will
+display forms according to targeting and behavior settings and collect delivery and engagement analytics automatically.
+
+### Prerequisites
+
+* Using Klaviyo SDK version 4.2.0 and higher
+* Imported `KlaviyoSwift` and `KlaviyoForms` SDK modules and adding it to the app target.
+
+### Setup
+
+Once you've created an in-app form in your Klaviyo account, to display it add the following code to your application.
```

Comment:
Can we simplify and just say "To display in-app forms add the following code to your application."

Reply:
```suggestion
To display in-app forms, add the following code to your application.
```